protocol-definitions/MC.yaml

```diff
@@ -725,4 +724,20 @@ methods:
           since: 2.0
           doc: |
             UUID of the consistency check request or null if consistency
-            check is ignored because of the configuration
\ No newline at end of file
+            check is ignored because of the configuration
+  - id: 24
+    name: pollMCEvents
+    since: 2.0
+    doc: |
+      Polls events available on member. Once read, events are removed from member's internal queue.
+    request:
+      retryable: true
```

Comment:
If request is not idempotent, it should not be retryable.

Reply:
Thanks for noticing this misconfiguration!

Yes, it's not idempotent. Going to fix it.